server/handlers/image_handler.py

```diff
@@ -1,19 +1,53 @@
+import logging
+import socket
 from random import random, randint, choice
 import string
 
 
 class ImageHandler:
-    def __init__(self, gs):
+    def __init__(self, gs, config):
+        self.logger = logging.getLogger("main")
         self.gs = gs
+        self.config = config
+        print("╠ CREATED IMAGE HANDLER")
+        self.logger.info("╠ CREATED IMAGE HANDLER")
 
+    def socket_connect(self):
+        while True:  # Restart the socket connection on disconnect
+            with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
+                s.bind((self.config["uav"]["images"]["host"], self.config["uav"]["images"]["port"]))
+                s.listen()
+                conn, addr = s.accept()
+                with conn:
+                    self.logger.critical("[Image] Socket connection established with %s:%s", *addr)
```

Comment:
I don't think this should be logged at the critical level. I would consider critical worse than error, implying something very bad has happened from which the program can't exactly recover. I think this should probably just be logged at the `info` level. Here's this article that's made some good points about it: https://www.sentinelone.com/blog/the-10-commandments-of-logging/.

Reply:
I definitely agree, and I did initially think of using `INFO`; however, I wanted this message to show up on the console (which only accepts messages from `WARNING` or above. So this was my little hack to make that work: change the meaning of the `CRITICAL` level to mean "Important Info". Since nothing in GroundStation uses the `CRITICAL` log level, I decided on this. It's not a perfect approach, but it worked so I used it.

Reply:
That's very hacky lol, but yeah I see why. You could add a logging level in between `info` and `warning`, and perhaps call it something like `IMPORTANT`. `INFO` is defined as 20, and `WARNING` is defined as 30, maybe make `IMPORTANT` 35.

Reply:
Good Idea!